apps/openmw/mwmechanics/stat.cpp

```diff
@@ -1,4 +1,5 @@
 #include "stat.hpp"
+#include "limits.h"
```

Comment:
Why don't you use the C++ way?

`#include <limits>` and `std::numeric_limits<int>::max()`